src/coreclr/jit/emitxarch.cpp

```diff
@@ -254,7 +254,19 @@ bool emitter::IsEvexEncodableInstruction(instruction ins) const
     {
         return false;
     }
-    return HasEvexEncoding(ins);
+
+    switch (ins)
+    {
+        case INS_pclmulqdq:
```

Comment:
Used a switch here and below because there will be more of these (VAES is set up the same way)